cmd/queue/main.go

```diff
@@ -262,9 +262,10 @@ func main() {
 		Handler: createAdminHandlers(),
 	}
 
-	timeoutHandler := queue.TimeToFirstByteTimeoutHandler(http.HandlerFunc(handler(reqChan, breaker, httpProxy)),
+	var composedHandler http.Handler = http.HandlerFunc(handler(reqChan, breaker, httpProxy))
+	composedHandler = queue.TimeToFirstByteTimeoutHandler(composedHandler,
 		time.Duration(revisionTimeoutSeconds)*time.Second, "request timeout")
-	composedHandler := pushRequestMetricHandler(pushRequestLogHandler(timeoutHandler))
+	composedHandler = pushRequestMetricHandler(pushRequestLogHandler(timeoutHandler))
```

Comment:
well, this is also flattenable, I guess?
Also add the same comment as in activator to what is the order of handler execution.

Reply:
:+1: